pkg/state/steps_uki.go

```diff
@@ -290,10 +320,10 @@ func (s *State) UkiPivotToSysroot(g *herd.Graph) error {
 				internalUtils.DropToEmergencyShell()
 			}
 
-			output, pcrErr := internalUtils.CommandWithPath("/usr/lib/systemd/systemd-pcrphase --graceful enter-initrd")
+			output, pcrErr := UKIExtendPCR()
```

Comment:
there is another call to the pcrphase at the end of uki, when jumping into systemd